plugins/plugin-java/che-plugin-java-plain/che-plugin-java-plain-ide/src/main/java/org/eclipse/che/plugin/java/plain/client/wizard/PlainJavaPagePresenter.java

```diff
@@ -145,7 +143,7 @@ private void setAttribute(String attrId, List<String> value) {
   }
 
   private boolean isCoordinatesCompleted() {
```

Comment:
This method looks redundant since there's just [one place](https://github.com/gazarenkov/che/blob/91392784e3fb6f87b9cb27e651de5c963990280e/plugins/plugin-java/che-plugin-java-plain/che-plugin-java-plain-ide/src/main/java/org/eclipse/che/plugin/java/plain/client/wizard/PlainJavaPagePresenter.java#L70-L73) where `isCoordinatesCompleted()` is called.